example.env

```diff
@@ -0,0 +1,17 @@
+# Reference: https://github.com/frappe/frappe_docker/blob/main/docs/images-and-compose-files.md
+
+FRAPPE_VERSION=v13.16.0
```

Comment:
How do we bump this with every release?

Reply:
We should. I create a script that would do that.

Reply:
Done: https://github.com/frappe/frappe_docker/pull/617/commits/adc526d8598120977d5d31874688e4057edc0b9c.